rotkehlchen/chain/ethereum/node_inquirer.py

```diff
@@ -263,11 +265,28 @@ def assure_curve_protocol_cache_is_queried(self) -> bool:
             return False
 
         curve_address_provider = self.contracts.contract('CURVE_ADDRESS_PROVIDER')
-        pools = query_curve_registry_pools(ethereum=self, curve_address_provider=curve_address_provider)  # noqa: E501
+        curve_address_provider = self.contracts.contract('CURVE_ADDRESS_PROVIDER')
```

Comment:
what are you doing here? You are reassigning the same variable by duplicating the line
```suggestion
```